myLivingDead/src/main/java/Main/Main.java

```diff
@@ -3,13 +3,134 @@
  */
 package Main;
+import Entity.Entity;
+import Entity.Survivor.Survivor;
 import Entity.Survivor.Type.*;
+import Entity.Zombie.Zombie;
 import Entity.Zombie.Type.*;
+import java.util.*;
 
 /**
  *
  * @author kaifa
  */
 public class Main {
     
+    private ArrayList<Survivor> survivorList = new ArrayList<>();
+    private ArrayList<Zombie> zombieList = new ArrayList<>();
+    
+    //create random number of survivors
+    public void createSurvivor(){
+        Random rand = new Random();
+        //generate numbers between 1 and 20
+        int numOfSurvivor = rand.nextInt(20)+1;
+        for(int i=0;i<numOfSurvivor;i++){
+            //generate numbers 0 to 2
+            int randNum = rand.nextInt(3);
+            switch (randNum) {
+                case 0://create Soldier survivor if random number is 0
+                    survivorList.add(new Soldier());
+                    break;
+                case 1://create Teacher survivor if random number is 1
+                    survivorList.add(new Teacher());
+                    break;
+                case 2://create Child survivor if random number is 2
+                    survivorList.add(new Child());
+                    break;
+            }
+        }
+    }
+    
+    //create random number of zombies
+    public void createZombie(){
+        Random rand = new Random();
+        //generate numbers between 1 and 10;
+        int numOfZombie = rand.nextInt(10)+1;
+        for(int i=0;i<numOfZombie;i++){
+            //generate numbers 0 to 1
+            int randNum = rand.nextInt(2);
+            switch (randNum) {
+                case 0://create Tank zombie if random number is 0
+                    zombieList.add(new Tank());
+                    break;
+                case 1://create CommonInfected zombie if random number is 1
+                    zombieList.add(new CommonInfected());
+                    break;
+            }
+        }
+    }
+    
+    //each survivor attacks every zombie
+    public void survivorAttack(){
+        for(int i=0;i<survivorList.size();i++){
+            for(int j=0;j<zombieList.size();j++){
+                //if survivor is alive, survivor attacks
+                if(survivorList.get(i).checkAlive()==true){
+                    survivorList.get(i).doAttack(zombieList.get(j));
+                }
+            }
+        }
+    }
+    
+    //each zombie attacks every survivor
+    public void zombieAttack(){
+        for(int i=0;i<zombieList.size();i++){
+            for(int j=0;j<survivorList.size();j++){
+                //if zombie is alive, zombie attacks
+                if(zombieList.get(i).checkAlive()==true){
+                    zombieList.get(i).doAttack(survivorList.get(j));
+                }
+            }
+        }
+    }
+    
+    //check for number of survivors alive and return number
+    public Integer checkSurvivingSurvivors(ArrayList<Survivor> list){
```

Comment:
I would recommend changing this method to take the an ArrayList<Entity> instead of ArrayList<Survivor> because then you could delete the second method checkSurvingZombies and just have one method checkSurvivingEntity that checks for the list of entities to see how many are still alive.